src/util.js

```diff
@@ -49,47 +49,30 @@ function handleVarlinkCallError(ex) {
 }
 
 export function updateContainers() {
-    let newContainers = {};
-    let newContainersStats = {};
-    return new Promise((resolve, reject) => {
-        varlink.call(PODMAN_ADDRESS, "io.podman.ListContainers")
-                .then(reply => {
-                    let newContainersMeta = reply.containers || [];
-                    let inspectRet = newContainersMeta.map(container => varlink.call(PODMAN_ADDRESS, "io.podman.InspectContainer", {name: container.id}));
```

Comment:
My impression was that this wanted to show much more information in the future. But we can bring it back if/when needed indeed, and maybe we'll only need it on a details page, not for all containers.